recipes/gargammel-slim/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.1.2" %}
+{% set sha256_gargammel = "c4b5ca081087fa5ecf086dbd49a4db0fb3c7d2d3d13eb30bdfc96b964661c3c1" %}
+{% set sha256_libgab = "ef4d4e980918f48b9a5c454f46afe3ffe6d97a075bbbf89abb7e3553cd6b9827" %}
+
+package:
+  name: gargammel-slim
+  version: {{ version }}
+
+build:
+  number: 0
+
+
+source:
+  - url: https://github.com/grenaud/libgab/archive/d2bbab7d88fbfdaa0ce57a26dc908d2fc66bf656.tar.gz
```

Comment:
seems a bit strange that this is not released... but i guess there's no harm to reproducibility with this snapshot